### PR TITLE
TASK: Set psalm baseline

### PR DESCRIPTION
**Pull request by kdambekalns**
This corrects some false positives

### CODE REVIEW

**Comment by kdambekalns** [comment]
Note: it does make a difference, which psalm version is used. I had 3.5.1 and could not reproduce the failures, with 3.5.3 I could. 🤷‍♂️

File: Neos.Flow/Classes/Property/TypeConverter/AbstractTypeConverter.php [hunk 1]
```diff
@@ -119,7 +119,7 @@ public function canConvertFrom($source, $targetType)
      * Returns an empty list of sub property names
      *
      * @param mixed $source
-     * @return array<string>
+     * @return array<mixed>
```
Comment (albe) [inline Neos.Flow/Classes/Property/TypeConverter/AbstractTypeConverter.php]: Is this really better than just `array`?

**Comment by albe** [comment]
> Note: it does make a difference, which psalm version is used. I had 3.5.1 and could not reproduce the failures, with 3.5.3 I could. 🤷‍♂️

Yeah, I think we need to pin psalm versions to make results more reproducible, then update more frequently.

File: Neos.Flow/Classes/Property/TypeConverter/AbstractTypeConverter.php (hunk 1 above)
Comment (kdambekalns) [reply Neos.Flow/Classes/Property/TypeConverter/AbstractTypeConverter.php]: No, but it is what the interface uses: https://github.com/neos/flow-development-collection/blob/50166ec3fa9dc13a92ba6b1babae6f99318d3cc3/Neos.Flow/Classes/Property/TypeConverterInterface.php#L78 🤷‍♂️

**Comment by kdambekalns** [comment]
> > Note: it does make a difference, which psalm version is used. I had 3.5.1 and could not reproduce the failures, with 3.5.3 I could. 🤷‍♂️
> 
> Yeah, I think we need to pin psalm versions to make results more reproducible, then update more frequently.

If we added it to the dev dist, it would update automatically… I just have it installed globally, to avoid changing the composer manifest "for no reason"…

File: Neos.Flow/Classes/Property/TypeConverter/AbstractTypeConverter.php (hunk 1 above)
Comment (albe) [reply Neos.Flow/Classes/Property/TypeConverter/AbstractTypeConverter.php]: Hmm. Okay then 🐑

**Pull request merged**